tests/web_client_specs/containerViewSpec.js

```diff
@@ -0,0 +1,159 @@
+/* globals girderTest, describe, it, expect, waitsFor, runs */
+girderTest.importPlugin('jobs', 'worker', 'slicer_cli_web');
+
+var slicer;
+girderTest.promise.done(function () {
+    slicer = girder.plugins.slicer_cli_web;
+});
+
+girderTest.startApp();
+
+function login(user, password) {
+    girderTest.waitForLoad('login wait 1');
+    runs(function () {
+        $('.g-login').click();
+    });
+
+    girderTest.waitForDialog('login wait 2');
+    runs(function () {
+        $('#g-login').val(user || 'user');
+        $('#g-password').val(password || 'password');
+        $('#g-login-button').click();
+    });
+
+    waitsFor(function () {
+        return $('.g-user-dropdown-link').length > 0;
+    }, 'user to be logged in');
+    girderTest.waitForLoad('login wait 3');
+}
+
+//Creates container and Default Task Folder and sets it in the settings using REST
+function createDefaultTaskFolder(name, description, public, folder){
+    var collectionID = false;
+    waitsFor(function () {
```

Comment:
Properly, this should be in a `runs` block, not a `waitsFor` block.  `waitFor` repeated executes until it returns true, which `runs` runs once.  In jasmine, generally we only use `waitsFor` to ensure asynchronous conditions are met.